.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  open-pull-requests-limit: 10
```

Comment:
Is there a reason you are setting this to 10 instead of 5 like in the [CLI](https://github.com/heroku/cli/blob/0d84d34184ae6a6c983484991fb29b61e384ee4c/.github/dependabot.yml#L9)?

Reply:
thanks for noticing that, reduced to 5.